javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/contexts/ObjectCreationContext.java

```diff
@@ -73,6 +74,9 @@ public ObjectCreationContext(ObjectCreationExpr wrappedNode, TypeSolver typeSolv
 
     @Override
     public SymbolReference<ResolvedMethodDeclaration> solveMethod(String name, List<ResolvedType> argumentsTypes, boolean staticOnly, TypeSolver typeSolver) {
+        //ResolvedType resolvedType = JavaParserFacade.get(typeSolver).getType(wrappedNode);
+        //List<ResolvedMethodDeclaration> possibleMethods = resolvedType.asReferenceType().getAllMethods();
+        //return MethodResolutionLogic.findMostApplicable(possibleMethods, name, argumentsTypes, typeSolver);
```

Comment:
Work in progress? :-)

Reply:
Committed by mistake together with Javadocs :( removing